src/webcomponents/clinical/clinical-analysis-create.js

```diff
@@ -247,25 +247,25 @@ export default class ClinicalAnalysisCreate extends LitElement {
     onClear() {
         this.initClinicalAnalysis();
         // This reset all date elements such as dueDate, check TASK-340
-        Array.from(this.querySelectorAll('input[type="date"]')).forEach(el => el.value = "");
+        Array.from(this.querySelectorAll("input[type='date']")).forEach(el => el.value = "");
         this.requestUpdate();
     }
 
     onSubmit() {
         // Prepare the data for the REST create
-        const data = {...this.clinicalAnalysis};
+        let data = {...this.clinicalAnalysis};
 
         // remove private fields
         delete data._users;
-
-        data.proband = {
-            id: this.clinicalAnalysis.proband?.id ? this.clinicalAnalysis.proband.id : null
-        };
+        data = {...data, proband: {id: this.clinicalAnalysis?.proband?.id ? this.clinicalAnalysis?.proband?.id : null}};
```

Comment:
I think this would be better to understand if you put it in several lines: 

```
data = {
    ...data,
    proband: {
        id: ...,
    },
};
```